modules/apps/transfer/ibc_module.go

```diff
@@ -291,3 +291,58 @@ func (im IBCModule) OnTimeoutPacket(
 
 	return nil
 }
+
+// OnChanUpgradeInit implements the IBCModule interface
+func (im IBCModule) OnChanUpgradeInit(
+	ctx sdk.Context,
+	order channeltypes.Order,
+	connectionHops []string,
+	portID, channelID string,
+	sequence uint64,
+	counterparty channeltypes.Counterparty,
+	version, previousVersion string,
+) (string, error) {
+	return version, nil
```

Comment:
shouldn't these just error for now?

Reply:
I thought we would just no-op for these at the moment rather than error?